index.html

```diff
@@ -3077,7 +3076,9 @@ <h3>Trust Model</h3>
 members of a community in which all members trust each other due to the
 rules of membership. Some [=verifiers=] might trust a specific trust-service
 provider whose responsibility is to vet [=issuers=] and list them in a trust
-list such as those specified in [[[ETSI]]] [[ETSI]].
+list such as those specified in "Electronic Signatures and Infrastructures 
+(ESI); Trusted Lists" [?ETSI] or the Adobe Approved Trust List Members 
+(https://helpx.adobe.com/acrobat/kb/approved-trust-list1.html).
```

Comment:
I wonder whether the AATL should also be added to `common.js`?

Don't reject the `[[[ETSI]]]` here, unless you see and disagree with its effect after being commit.
```suggestion
list such as those specified in [[[ETSI]]] [[?ETSI]] or the
<a href="https://helpx.adobe.com/acrobat/kb/approved-trust-list1.html">Adobe
Approved Trust List</a>.
```